src/modules/submit.js

```diff
@@ -0,0 +1,14 @@
+import { submitScore } from './lapi.js';
+import { refresh } from './reload.js';
+
+const submitEven = async (e) => {
+  e.preventDefault();
+  const form = document.getElementById('form');
+  const name = form.elements.name.value;
+  const score = form.elements.score.value;
+  await submitScore(name, score);
+  refresh();
+  form.reset();
+};
```

Comment:
- [ ] While trying to submit a new score to the leaderboard, there is an error happen (see the image below), Please fix that.

![image](https://user-images.githubusercontent.com/68796232/219884325-95b00431-8e2b-4a80-8194-3040f7edd796.png)